### DISCUSSION TITLE
Cant drag and drop images

### DISCUSSION THREAD

**Discussion by Honkebiiib** [lllyasviel/Fooocus#1287] [Q&A]
Hi,

i cant drag and drop rendered images into the inpaint panel for example or the img2img panel. i first have to save the image and drag in from the explorer.
any advice for that?

**Comment by ludovitkapusta** [comment]
its not working for imageprompt ... so you have to do it manualy from computer

btw, all generated images are in Foocus/outputs

**Comment by ErmMonzon** [comment]
Have you tried it on Chrome? Works for me. Firefox doesn't work for some reason.

**Comment by Honkebiiib** [comment]
Chrome works fine yes!
